### ISSUE TITLE
Error deploying to Google Cloud Platform - lscpu: failed to determine number of CPUs

### ISSUE THREAD

**Issue by karltaylor** [parcel-bundler/parcel#2019]
# 🐛 bug report

I cannot deploy to App Engine on Google Cloud Platform because of the following error:

```
lscpu: failed to determine number of CPUs: /sys/devices/system/cpu/possible: No such file or directory
```

## 🎛 Configuration (.babelrc, package.json, cli command)

I am using the parcel middleware to run parcel alongside Express:

```js
const express = require("express");
const app = express();
const Bundler = require("parcel-bundler");

const file = "./app/views/index.pug";

const bundler = new Bundler(file, options);

app.use(bundler.middleware());

app.listen(PORT, () =>
  console.log(`App running locally: http://localhost:${PORT}`)
);
```

## 🤔 Expected Behavior

Parcel app should deploy as normal.

## 😯 Current Behavior

The GCP logs say:

```
lscpu: failed to determine number of CPUs: /sys/devices/system/cpu/possible: No such file or directory
```

`"parcel-bundler": "^1.9.7"`

**Comment by karltaylor** [comment]
Related: #133

**Comment by DeMoorJasper** [comment]
This error should be caught, this should only be a warning log?

https://github.com/parcel-bundler/parcel/blob/master/src/utils/cpuCount.js

**Comment by karltaylor** [comment]
@DeMoorJasper, it looks like it is something other than this error.

Unfortunately I get a 500 error when using the parcel bundle middleware with express.

However if I deploy a simple `get` req everything is fine, so It might be something to do with Google Cloud Platform and the Parcel Bundler. 

```js
app.get('/', (req, res) => {
  res.send('hello world') //- works fine
}
```

```js
app.use(bundler.middleware()); //- returns 500
```

**Comment by DeMoorJasper** [comment]
If it's a 500 than it should log some kind of error

**Comment by karltaylor** [comment]
Sorry about this, you're correct.

I know it's not related and it's going far off topic now, but:

For GCP App Engine I needed to add a bigger instance class because it was running out of memory: 

```js
// app.yaml
runtime: nodejs8

instance_class: F2
```

After this, I got the following error from Google Cloud Platform: App Engine.

```bash
Error writing to cache: EROFS: read-only file system
```

And after doing some digging I came across [this stackoverflow post about not being able to write files](https://stackoverflow.com/questions/2693081/does-google-app-engine-allow-creation-of-files-and-folders-on-the-server).

So in the end I have just changed my express `server.js` to serve the `dist/index.html` file

```js
app.get('/', (req, res) => {
  res.sendFile('/dist/index.html')
})
```

and in my `package.json`:

```json
  "scripts": {
    "start": "node index.js",
    "bundle": "parcel build app/views/index.pug",
    "prestart": "npm run bundle"
}
```

And now I finally have it working on google cloud platform! 

It currently does not working using the express middleware approach though.
